src/BigQueryClientHandler.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BigQueryTransformation;
+
+use DateTime;
+use GuzzleHttp\Client;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\MessageFormatter;
+use GuzzleHttp\Middleware;
+use JsonException;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Psr\Log\NullLogger;
+
+class BigQueryClientHandler
```

Comment:
Udělal sem nový handler pro guzzle, dost důležitý je retry na RETRY_MISSING_CREATE_JOB protože to sa objeví když sa nespropagujů IAM práva při vytvoření WS.